src/components/Channel/Channel.tsx

```diff
@@ -1346,6 +1327,10 @@ const ChannelInner = <
     );
   }
 
+  // @ts-ignore
+  window['channel'] = channel;
+  // @ts-ignore
+  window['client'] = client;
```

Comment:
Let's make sure we don't merge this code :)

Reply:
true :)